src/System.Private.ServiceModel/tools/scripts/RemoveFirewallPorts.cmd

```diff
@@ -0,0 +1,39 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+REM
+echo Start Remove all ports WCF test services use
+REM
+
+SET __EXITCODE=0
+
+netsh advfirewall firewall delete rule name="_WCF Test Server PortHttp"
+call :CheckError %ERRORLEVEL% "_WCF Test Server PortHttp"
+
+netsh advfirewall firewall delete rule name="_WCF Test Server PortHttps"
+call :CheckError %ERRORLEVEL% "_WCF Test Server PortHttps"
+
+netsh advfirewall firewall delete rule name="_WCF Test Server PortTcp"
+call :CheckError %ERRORLEVEL% "_WCF Test Server Port
+
+netsh advfirewall firewall delete rule name="_WCF Test Server PortWebSocket"
+call :CheckError %ERRORLEVEL% "_WCF Test Server PortWebSocket"
+
+netsh advfirewall firewall delete rule name="_WCF Test Server PortWebSockets"
+call :CheckError %ERRORLEVEL% "_WCF Test Server PortWebSockets"
+
+goto :done
+
+:CheckError
+SET _ErrorLevel=%1
+if NOT [%_ErrorLevel%]==[0] if NOT [%_ErrorLevel%]==[1] (
```

Comment:
I notice this pattern a lot, but it's not apparent to me in any of the scripts what ERRORLEVEL == 1 is. 0 is pretty standard, but if 1 is something special in netsh, it'd be great to put some comments here to explain why 1 is an acceptable errorlevel to ignore when calling netsh.